### PR TITLE
Various minor code cleanups

### PR DESCRIPTION
**Pull request by nicolas-grekas**
In case this might be of interest to you.

### CODE REVIEW

File: tests/ProxyManagerTestAsset/ClassWithFinalMethods.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ class ClassWithFinalMethods extends TestCase
     {
     }
 
-    final private function bar()
+    private function bar()
```
Comment (nicolas-grekas) [inline tests/ProxyManagerTestAsset/ClassWithFinalMethods.php:21]: triggers a warning on PHP 8

**Review by Ocramius — APPROVED** [review]
Excellent, thanks @nicolas-grekas!

**Comment by Ocramius** [comment]
Ah, the failures are mostly on generated code expectations...

**Comment by nicolas-grekas** [comment]
> Ah, the failures are mostly on generated code expectations...

yep, updated and green now.

**Comment by Ocramius** [comment]
Great! Thank you \o/

**Comment by Ocramius** [comment]
Hmm, I see that release automation failed to move the active branch to `2.10.x` - up to me to fix that or release this under `2.9.2`

**Pull request merged**